win32/Lib/pywin32_testutil.py

```diff
@@ -62,28 +65,30 @@ def countTestCases(self):
         return self.num_test_cases
 
     def __call__(self, result=None):
-        # For the COM suite's sake, always ensure we don't leak
-        # gateways/interfaces
-        from pythoncom import _GetInterfaceCount, _GetGatewayCount
-
-        gc.collect()
-        ni = _GetInterfaceCount()
-        ng = _GetGatewayCount()
+        if not IS_PYPY:
+            # For the COM suite's sake, always ensure we don't leak
+            # gateways/interfaces
+            from pythoncom import _GetInterfaceCount, _GetGatewayCount
+
+            gc.collect()
+            ni = _GetInterfaceCount()
+            ng = _GetGatewayCount()
         self.real_test(result)
         # Failed - no point checking anything else
         if result.shouldStop or not result.wasSuccessful():
             return
         self._do_leak_tests(result)
-        gc.collect()
-        lost_i = _GetInterfaceCount() - ni
-        lost_g = _GetGatewayCount() - ng
-        if lost_i or lost_g:
-            msg = "%d interface objects and %d gateway objects leaked" % (
-                lost_i,
-                lost_g,
-            )
-            exc = AssertionError(msg)
-            result.addFailure(self.real_test, (exc.__class__, exc, None))
+        if not IS_PYPY:
+            gc.collect()
+            lost_i = _GetInterfaceCount() - ni
+            lost_g = _GetGatewayCount() - ng
+            if lost_i or lost_g:
+                msg = "%d interface objects and %d gateway objects leaked" % (
+                    lost_i,
+                    lost_g,
+                )
+                exc = AssertionError(msg)
+                result.addFailure(self.real_test, (exc.__class__, exc, None))
```

Comment:
You're voluntarily causing leaks in tests here. At this point, better just disable the test entirely on PyPy